notebooks/lesson09-overfitting.ipynb

```diff
@@ -0,0 +1,2113 @@
+{
```

Comment:
<ul><li>i would not say <code>train/test split</code> yet since i've only been talking about training and validations sets and they might get confused</li><li>typo: "... knew that fitting complex models ..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>